frontend/src/app/components/chat-input/chat-input.component.ts

```diff
@@ -26,7 +27,7 @@ export class ChatInputComponent {
 						displayName: "You",
 						isCurrentUser: true,
 						userIsDeleted: false,
-						messageText: this.newMessage.value!,
+						messageText: this.newMessage.value!.length > this.limit ? this.newMessage.value!.substring(0,this.limit) : this.newMessage.value!,
```

Comment:
May you please format the code for better readability:

```suggestion
						messageText: this.newMessage.value!.length > this.limit 
						    ? this.newMessage.value!.substring(0,this.limit) 
						    : this.newMessage.value!,
```